src/WebJobs.Script.WebHost/Diagnostics/MetricsEventManager.cs

```diff
@@ -567,6 +584,8 @@ public RunningFunctionInfo(string name, Guid invocationId, DateTime startTime, b
                 public DateTime EndTime { get; set; }
 
                 public bool Success { get; set; }
+
+                public string TriggerType { get; private set; }
```

Comment:
Remind me what you're going to use this for again?

Reply:
For tracking scale requirements per triggertype instead of at the app level.